<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorSpectrumPicker</class>
 <widget class="QWidget" name="ColorSpectrumPicker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>229</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ColorSpectrumChannelPicker" name="pickerRed" native="true"/>
   </item>
   <item>
    <widget class="ColorSpectrumChannelPicker" name="pickerGreen" native="true"/>
   </item>
   <item>
    <widget class="ColorSpectrumChannelPicker" name="pickerBlue" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorSpectrumChannelPicker</class>
   <extends>QWidget</extends>
   <header>colorspectrumchannelpicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
